tests/fixture/MiG-users.db--example.binary

```diff
@@ -0,0 +1,187 @@
+(dp0
+V/C=DK/ST=NA/L=NA/O=Test Org/OU=NA/CN=b'Test User'/emailAddress=dummy-user
+p1
+(dp2
+Vfull_name
+p3
+Vb'Test User'
+p4
+sVorganization
+p5
+VTest Org
+p6
+sVstate
+p7
+VNA
+p8
+sVcountry
+p9
+VDK
+p10
+sVemail
+p11
+Vdummy-user
+p12
+sVcomment
+p13
+VThis is the create comment
+p14
+sVpassword
+p15
+V
+p16
+sVpassword_hash
+p17
+VPBKDF2$sha256$10000$b't0JM/JjkQ347th0Q'$b'QupJt53hA5KhESEeqDhTQTCPOrCBvZ6H'
+p18
+sVdistinguished_name
+p19
+V/C=DK/ST=NA/L=NA/O=Test Org/OU=NA/CN=b'Test User'/emailAddress=dummy-user
+p20
+sVlocality
+p21
+g16
+sVorganizational_unit
+p22
+g16
+sVexpire
+p23
+I1757925298
+sVcreated
+p24
+F1726233828.2676349
+sVunique_id
+p25
+VktyCKIRg9HvsVzXMQ22EaKS67t9atchv9JKTiJqrtBiGN3qksKrbTTYIH8mitY2K
+p26
+sVopenid_names
+p27
+(lp28
+sVold_password_hash
+p29
+VPBKDF2$sha256$10000$b'GL7Qq92iLe/hZXBo'$b'ZwB/5IZqgU7onP+ZqZk9zcHVZOx7jmWz'
+p30
+sVrenewed
+p31
+F1726389298.7801197
+ssV/C=DK/ST=NA/L=NA/O=Test Org/OU=NA/CN=Test User/emailAddress=dummy-user
+p32
+(dp33
+Vfull_name
+p34
+VTest User
+p35
+sVorganization
+p36
+VTest Org
+p37
+sVstate
+p38
+VNA
+p39
+sVcountry
+p40
+VDK
+p41
+sVemail
+p42
+Vdummy-user
+p43
+sVcomment
+p44
+VThis is the create comment
+p45
+sVpassword
+p46
+g16
+sVpassword_hash
+p47
+VPBKDF2$sha256$10000$b'kZ8WgLNH+wg3X11d'$b't1d08MV4g215WYW7S7EbkjHqDF+MCjMa'
```

Comment:
off-topic: this incidentally looks like a buggy PBKDF hash construction. That `b''` most definitely should not be there.
Not sure it is a result from mixing python3 and `edge` or a real bug though.

Reply:
I've definitely seen this before and I thought even addressed it - see: https://github.com/ucphhpc/migrid-sync/pull/117

That PR strictly just made this _work_ on PY3 and now I read it again it seems it replicated the same bug you've noticed here - which at minimum is interesting. Unfortunately it points to potentially rather worrying situation: we may have written user DBs on PY3 with subtly screwy password hashes and if we fixed this we may risk that they attempts to validate the hashes would fail. Without such a fix a PY2 user DB will not operate on PY3. :/

Reply:
Alright, thanks for elaborating.
This kind of subtle _silent_ error is one of the main remaining worries for the migration of sites that started out on py2 IMO. I think we're otherwise in a quite good place with all the testing and verification work work done.
Definitely something to look more into and carefully test both pre- and post-migration.